R.swift/Generators/SegueGenerator.swift

```diff
@@ -41,19 +41,31 @@ struct SegueGenerator: Generator {
       .values
       .flatMap { $0.first }
 
-    let groupedSeguesWithInfo = deduplicatedSeguesWithInfo
-      .groupUniquesAndDuplicates { "\(sanitizedSwiftName($0.segue.identifier))|\($0.sourceType)" }
+    var structs: [Struct] = []
 
-    for duplicate in groupedSeguesWithInfo.duplicates {
-      let anySegueWithInfo = duplicate.first!
-      let names = duplicate.map { $0.segue.identifier }.sort().joinWithSeparator(", ")
-      warn("Skipping \(duplicate.count) segues for '\(anySegueWithInfo.sourceType)' because symbol '\(sanitizedSwiftName(anySegueWithInfo.segue.identifier))' would be generated for all of these segues, but with a different destination or segue type: \(names)")
-    }
+    for (sourceType, seguesBySourceType) in deduplicatedSeguesWithInfo.groupBy({ $0.sourceType }) {
+      let gropuedSeguesWithInfo = seguesBySourceType.groupBySwiftNames { $0.segue.identifier }
 
-    let structs = groupedSeguesWithInfo.uniques
-      .groupBy { $0.sourceType }
-      .values
-      .flatMap(SegueGenerator.seguesWithInfoForSourceTypeToStruct)
+      for (name, duplicates) in gropuedSeguesWithInfo.duplicates {
+        warn("Skipping \(duplicates.count) segues for '\(sourceType)' because symbol '\(name)' would be generated for all of these segues, but with a different destination or segue type: \(duplicates.joinWithSeparator(", "))")
+      }
+
+      let empties = gropuedSeguesWithInfo.empties
+      if let empty = empties.first where empties.count == 1 {
+        warn("Skipping 1 segue for '\(sourceType)' because no swift identifier can be generated for segue: \(empty)")
```

Comment:
Would be a nice future improvement I think. It wouldn't be a warning that you get a lot of at once.